### DISCUSSION TITLE
Attach/detach for :through associations

### DISCUSSION THREAD

**Discussion by segiddins** [avo-hq/avo#1798] [Feature feedback]
It doesn't make sense to me that attach/detach/delete are available on associations that are `:through` another association, and it seems like "detaching" one of those associations actually destroys the intermediate object?

**Comment by adrianthedev** [comment]
It could be that the buttons are there just because the fields use the regular non-`through` (`has_many`, `has_one`) field components.
I think they can be hidden using the authorization policies, right?

**Comment by segiddins** [comment]
Yeah they can, but it feels a bit to me like avo is doing a dangerous/unexpect thing by default here
